composer.json

```diff
@@ -4,7 +4,7 @@
     "keywords": ["task-runner", "changelog"],
     "type": "library",
     "require": {
-        "php": "~7.1",
+        "php": ">=7.2.0",
         "openeuropa/task-runner": "dev-master"
```

Comment:
We have circular dependency with this. See: https://github.com/openeuropa/task-runner/blob/9a688023c8f485b0e8004529f4f7b5f5fc550083/composer.json#L15